packages/grafana-ui/src/components/Flex/Flex.internal.story.tsx

```diff
@@ -0,0 +1,209 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { Text } from '../../components/Text/Text';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+
+import { Flex } from './Flex';
+import mdx from './Flex.mdx';
+
+const Item = ({ color, text, height }: { color: string; text?: string | number; height?: string }) => {
+  return (
+    <div
+      style={{
+        width: '5em',
+        height: height,
+        backgroundColor: color,
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+    >
+      {text && (
+        <Text color="info" variant="h2">
+          {text}
+        </Text>
+      )}
+    </div>
+  );
+};
+
+const meta: Meta<typeof Flex> = {
+  title: 'General/Flex',
+  component: Flex,
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Basic: StoryFn<typeof Flex> = ({ direction, wrap, alignItems, justifyContent, gap }) => {
+  return (
+    <Flex direction={direction} wrap={wrap} alignItems={alignItems} justifyContent={justifyContent} gap={gap}>
```

Comment:
Lets put the basic in some sort of container with a height and width so that we can test the align and justify properly. We can add a border to that container as well so people can properly see where it is